kit/src/elements/button/mod.rs

```diff
@@ -79,6 +79,8 @@ pub fn Button<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
         }
     );
 
+    let show_icon = cx.props.loading.unwrap_or(false) || cx.props.icon.is_some();
```

Comment:
a boolean default will always be false

```suggestion
    let show_icon = cx.props.loading.unwrap_or_default() || cx.props.icon.is_some();
```